api/src/controllers/webfinger.js

```diff
@@ -133,7 +133,8 @@ function WebfingerControllerFactory(log, config, ledger) {
       this.body = {
         'subject': config.data.get('client_host'),
         'properties': {
-          'https://interledger.org/rel/publicKey': config.data.getIn(['connector', 'public_key'])
+          'https://interledger.org/rel/publicKey': config.data.getIn(['connector', 'public_key']),
+          'https://interledger.org/rel/protocolVersion': 'Compatible: ilp-kit v' + config.data.getIn(['ilpKitVersion'])
```

Comment:
I put the word 'Compatible' there because people may be running ledgers and/or connectors that are not ilp-kit.

It would therefore be a bit weird to announce you're compatible with a specific commit of ilp-kit master; if there is a need for that (e.g. a known issue is fixed) then we can also just tag versions more often?